README.md

```
@@ -46,10 +46,12 @@ team-project/
 
 
 ## 수정 이력 테이블
-| 번호 | 작업 날짜 | 작업 내용 |
-|:---:|:---:|:---:|
-| 1 | 2026.03.18 15:18 | README.md 작성 |
-| 2 | 2026.03.18 16:07 | member-lee 작성 |
+| 번호 |      수정 날짜       |       수정 내용       | 작업자  |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)